pages/cockpit/user-guide/manage-organizations.adoc

```diff
@@ -31,6 +32,11 @@ image:cockpit/organization-settings.png[]
 . Click *SAVE*.
 . Update the *Human Readable ID* and click *CHANGE HRID*.
 
+== Delete the organization
+
+. Go to the organization settings page.
+. Click *DELETE THIS ORGANIZATION*. 
```

Comment:
Maybe we should add a note to explain that an organization can not be deleted if it has environments